src/ProductConstructionService/ProductConstructionService.Api/Controllers/ServiceStatusController.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace ProductConstructionService.Api.Controllers;
+
+[ApiController]
+[Route("status")]
+public class ServiceStatusController : Controller
+{
+    [HttpGet("startup")]
```

Comment:
Should we just use the `/health` one provided by ASP.NET?

Reply:
I think we'll want to have a specialized one for the service, that we can configure to say `ready` after initializing the VMR, for example

Reply:
Let's add it then. It might happen quite far in the future or not at all and until then, I think we might be missing out on whatever the health endpoint does - I imagine it might be smart but don't know - would be nice to find out maybe

Reply:
Also - can there be more? Maybe we keep the default and just add a second one later?

Reply:
Yeah, that might be the best for now, I'll delete this controller, and set the Health probe to use the `/health` endpoint.